app/pb_kits/playbook/pb_progress_simple/_progress_simple.jsx

```diff
@@ -1,21 +1,74 @@
+/* @flow */
 import React from 'react'
-import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { buildCss } from '../utilities/props'
 
-const propTypes = {
-  className: PropTypes.string,
-  id: PropTypes.string,
+type ProgressSimpleProps = {
+  align?: "left" | "center" | "right",
+  className?: String | Array<String>,
+  dark?: Boolean,
+  data?: String,
+  id?: String,
+  max: String,
+  muted: Boolean,
+  percent: String,
+  value: Number,
+  width: String
 }
 
-class ProgressSimple extends React.Component {
-  render() {
-    return (
-      <div className="pb_progress_simple">
-        <span>{'PROGRESS SIMPLE CONTENT'}</span>
-      </div>
-    )
+const ProgressSimple = ({
+  align,
+  className,
+  dark = false,
+  max,
+  muted = false,
+  percent,
+  value,
+  width = '100%',
+}: ProgressSimpleProps) => {
+  const styles = {
+    width: width,
   }
-}
 
-ProgressSimple.propTypes = propTypes
+  const valueStyles = {
+    width: `${percent}%`,
+  }
+
+  const nonPercentStyles = {
+    width: `${(value * 100) / max}%`,
+  }
+
+  const wrapperClass = classnames(
+    className,
+    buildCss('pb_progress_simple_wrapper', align, { dark: dark })
+  )
+
+  const kitClass = classnames(
+    className,
+    buildCss('pb_progress_simple_kit', align, { muted: muted })
+  )
+
+  return (
+    <div className={wrapperClass}>
+      <div
+          className={kitClass}
+          data-value={value}
+          style={styles}
+      >
+        <If condition={percent}>
+          <div
+              className="progress_simple_value"
+              style={valueStyles}
+          />
+          <Else />
+          <div
+              className="progress_simple_value"
+              style={nonPercentStyles}
+          />
+        </If>
```

Comment:
```suggestion
        <div
              className="progress_simple_value"
              style={valueStyles}
         />
```

Can we consolidate `valueStyles` and `nonPercentStyles` into one variable that uses `percent` to determine the output here? (see above suggestion)